odoo/modules/loading.py

```diff
@@ -31,56 +31,36 @@
     from odoo.tests.result import OdooTestResult
     from .module_graph import ModuleNode
 
+    LoadMode = typing.Literal['init', 'update']
```

Comment:
NIT: I am OK to have this typing here. But it would be ideal if we could have it in the `tools.convert_file` for `mode`. Since that function can be used outside of orm, developers can have a clear type hint.